be/src/olap/rowset/segment_v2/inverted_index_cache.cpp

```diff
@@ -179,6 +179,19 @@ Status InvertedIndexSearcherCache::erase(const std::string& index_file_path) {
     return Status::OK();
 }
 
+void InvertedIndexSearcherCache::prune() {
+    if (_cache) {
+        _cache->prune();
```

Comment:
`prune()` will free the entire cache, expecting it to be like `segment cache`,
During minor gc, prune_if evict the cache value with last_visit_time > N min
During full gc, prune evict entire cache